R/oauth-flow-auth-code.R

```diff
@@ -201,7 +212,15 @@ oauth_flow_auth_code <- function(client,
   )
   utils::browseURL(user_url)
 
-  if (redirect$localhost) {
+  if (redirect$can_fetch_code) {
+    # Wait a bit to give the user a chance to click through the authorisation
+    # process.
+    if (!is_testing()) {
```

Comment:
The fact that we need to do this to make the tests faster makes me think polling would be the better way to go.